staging/cse/windows/README

```diff
@@ -55,6 +55,9 @@ popd
 # AKS Windows CSE Scripts Package
 All files except *.test.ps1 and README will be published in AKS Windows CSE Scripts Package.
 
+## v0.0.35
+feat: update cse script package changes for aks windows gpu feature #3794
```

Comment:
v0.0.35 should also include this commit https://github.com/Azure/AgentBaker/commit/1c875831b2c7119c233510e8e60e374ad4d74bd2

`feat: optimize resize-osdrive replacing get-partitionsupportedsize with diskpart #3804`

Reply:
With checking the history of the folder `staging/cse/windows`
https://github.com/Azure/AgentBaker/commits/master/staging/cse/windows